### PR TITLE
add details component

### PR DESCRIPTION
**Pull request by aghh1504**

### CODE REVIEW

File: src/rsg-components/Markdown/Detail/Detail.spec.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+import Detail from './index';
```
Comment (sapegin) [inline src/rsg-components/Markdown/Detail/Detail.spec.js]: Please rename it back to `Details` to match the HTML element.

File: src/rsg-components/Markdown/Detail/Detail.spec.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+describe('Markdown Detail', () => {
+	it('should render a Detail', () => {
+		const actual = render(<Detail />);
```
Comment (sapegin) [inline src/rsg-components/Markdown/Detail/Detail.spec.js]: This component has required `children` prop, we need to test it.

File: src/rsg-components/Markdown/Detail/index.js [hunk 3]
```diff
@@ -0,0 +1,2 @@
+export { default } from 'rsg-components/Markdown/Detail/DetailRenderer';
```
Comment (sapegin) [inline src/rsg-components/Markdown/Detail/index.js]: Let's use named export for both components.

**Comment by aghh1504** [comment]
screen shot 2018-08-29 at 12 38 34

**Comment by aghh1504** [comment]
screen shot 2018-08-29 at 13 48 46

**Comment by aghh1504** [comment]
screen shot 2018-08-29 at 14 04 32

**Comment by aghh1504** [comment]
screen shot 2018-08-29 at 21 51 17

**Comment by aghh1504** [comment]
screen shot 2018-08-29 at 21 51 01

**Comment by aghh1504** [comment]
screen shot 2018-08-29 at 21 53 17

**Comment by aghh1504** [comment]
screen shot 2018-08-29 at 21 53 30

**Comment by sapegin** [comment]
Thanks, merging! 🤗

**Pull request merged**